scripts/sharedFuncs.sh

```diff
@@ -183,6 +183,7 @@ function download_component() {
             show_message "downloading $4 ..."
             ariapkg=$(package_installed aria2c "summary")
             curlpkg=$(package_installed curl "summary")
```

Comment:
I think we can opt out of curl, so remove any lines which use curl, such as this